integration/container/pause_test.go

```diff
@@ -62,7 +63,7 @@ func TestPauseFailsOnWindowsServerContainers(t *testing.T) {
 	poll.WaitOn(t, container.IsInState(ctx, client, cID, "running"), poll.WithDelay(100*time.Millisecond))
 
 	err := client.ContainerPause(ctx, cID)
-	assert.Check(t, is.ErrorContains(err, "cannot pause Windows Server Containers"))
+	assert.Check(t, is.ErrorContains(err, containerderrdefs.ErrNotImplemented.Error()))
```

Comment:
w.r.t. https://github.com/moby/moby/pull/42164/files#r602423617

The daemon may be converting it from the containerd error type; https://github.com/moby/moby/blob/81dbed4c8b87d99824eecea68c95f10056fd3d2a/errdefs/http_helpers.go#L186-L187

And on the client side, will handle it as an `errdefs.InotImplemented()`; https://github.com/moby/moby/blob/81dbed4c8b87d99824eecea68c95f10056fd3d2a/errdefs/is.go#L79-L83


Oh, actually, it may not be converted; looking at https://github.com/moby/moby/blob/7b9275c0da707b030e62c96b679a976f31f929d3/daemon/pause.go#L41-L43

It just trumps the error-type, and converts it into a generic error (likely will return a `500` error at the API level)